repos/spack_repo/builtin/packages/moab/package.py

```diff
@@ -208,6 +224,13 @@ def configure_args(self):
         else:
             options.append("--enable-fortran")
 
+        if "+tempest" in spec:
```

Comment:
(minor) This is considered to be more idiomatic:
```suggestion
        if spec.satisfies("+tempest"):
```